modules/custom-css/custom-css.php

```diff
@@ -802,6 +802,7 @@ static function admin() {
 				<div id="poststuff">
 					<p class="css-support"><?php echo apply_filters( 'safecss_intro_text', __( 'New to CSS? Start with a <a href="http://www.htmldog.com/guides/cssbeginner/">beginner tutorial</a>. Questions?
 		Ask in the <a href="http://wordpress.org/support/forum/themes-and-templates">Themes and Templates forum</a>.', 'jetpack' ) ); ?></p>
+					<p class="css-support"><?php echo __( 'Note: Custom CSS changes will reset when changing themes.', 'jetpack' ); ?></p>
```

Comment:
Something about this wording feels awkward.

Maybe "Custom CSS will be reset when changing themes." might be better?  Thoughts?